### PR TITLE
Edge: extract the common functions and put in one place for Edge test…

### PR DESCRIPTION
**Pull request by litian1992**
This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

File: test/cases/ostree-common-functions.sh [hunk 1]
```diff
@@ -0,0 +1,257 @@
+#!/bin/bash
+# A script that's full of common functions
+# used throughout our ostree tests. Can be 
+# sourced at beginning of those scripts.
+
+function CommonInit() {
```
Comment (mvo5) [inline test/cases/ostree-common-functions.sh:6]: Thanks for extracting common helper, that is super nice!

(super nitpick) maybe uses common_init as the name as everything else in this file uses snake_case ?

Comment (litian1992) [reply test/cases/ostree-common-functions.sh]: Thank you for the advice. I had it changed to snake_case.

**Review by henrywang — DISMISSED** [review]
LGTM. Thanks.

**Comment by litian1992** [comment]
Hello @ondrejbudai Can you please help me merge this? It passed all Edge tests. And the failed ones are irrelevant to this PR, I think. Thank you!

**Comment by bcl** [comment]
> Hello @ondrejbudai Can you please help me merge this? It passed all Edge tests. And the failed ones are irrelevant to this PR, I think. Thank you!

Currently there are conflicts with newer changes to osbuild-composer, you need to rebase and resolve those first.

**Pull request closed without merging**